ios/browser/ui/browser_menu/features.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_IOS_BROWSER_UI_BROWSER_MENU_FEATURES_H_
+#define BRAVE_IOS_BROWSER_UI_BROWSER_MENU_FEATURES_H_
+
+#import "base/feature_list.h"
+
+namespace brave::features {
```

Comment:
nit: Not sure if we do this anywhere else as it's C++17 iirc. I'm all for it, but usually I think we do nested namespaces?

```c++
namespace brave {
namespace features {

  //...........
  
}  //  features
}  // brave
```

Ignore if I'm incorrect.

Reply:
I've seen it done both ways in core, so I just picked this one (e.g. https://github.com/brave/brave-core/blob/1795aa0b6d519dc9806232e89c285ca6b15d13b3/components/ai_chat/core/common/features.h#L15), though after I wrote this originally it seems like we did have a discussion about nested namespaces in general https://bravesoftware.slack.com/archives/C7VLGSR55/p1732569420404999

It seems like the general consensus is that this is allowed for feature flags but can always remove it if we decide differently later